File: control-assurance-sp/src/types/GoMiscFile.ts
import { IEntity } from "./Entity";
import { User } from "./User";

export interface IGoMiscFile  extends IEntity {
    Details?: string;
    DateUploaded?: Date;
    UploadedByUserId?: number;
    User?: User;

}

export class GoMiscFile implements IGoMiscFile{ 
    public ID: number = 0;
    public Title: string = null;
    public Details?: string = null;
    public DateUploaded?: Date = null;
    public UploadedByUserId?: number = null;
    public User?: User = null; 
}

File: ControlAssuranceAPI/Models/GoMiscFile.cs
using System;
using System.Collections.Generic;

namespace CAT.Models;

public partial class GoMiscFile
{
    public int ID { get; set; }

    public string? Title { get; set; }

    public string? Details { get; set; }

    public DateTime? DateUploaded { get; set; }

    public int? UploadedByUserId { get; set; }

    public virtual User? User { get; set; }
}
